### ISSUE TITLE
Add term enlarged radial head

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10373]
**Preferred term label:**
Enlarged radial head

**Synonyms**
Large radial head

**Definition (free text, please give PubMed ID)**
Enlarged head of the radius

**Parent term (use hpo.jax.org/app)**
Abnormality of the radial head HP:0003995

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ACROCAPITOFEMORAL DYSPLASIA; ACFD, OMIM607778 (PMID: 12624140)

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
Adding new term: 
- ID: HP:6000885
- label: Enlarged radial head
- parents: [Abnormality of the radial head (HP:0003995)](https://hpo.jax.org/app/browse/term/HP:0003995)
- pmid: PMID:12624140
- def: Size (volume) of the head of the radius above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10373
- synonyms:
